entity_gym/entity_gym/environment.py

```diff
@@ -100,35 +100,35 @@ class Observation:
 @dataclass
 class CategoricalActionMaskBatch:
     actors: RaggedBufferI64
+    masks: Optional[RaggedBufferI64]
```

Comment:
This should use a bool or bitvec type but that doesn't exist yet so more of a note to myself.

Reply:
Yes I did think about this, but couldnt see that type in the raggedbuffer classes.

let me know when thats implemented and we can push it here.